nixos/attestation-server/module.nix

```diff
@@ -107,7 +107,9 @@ in
       wantedBy = [ "multi-user.target" ];
       requires = [ "network-online.target" ];
 
-      serviceConfig = {
+      serviceConfig = (lib.optionalAttrs (cfg.email.passwordFile != null) {
+        LoadCredential = lib."emailPassword:${cfg.email.passwordFile}";
```

Comment:
Confused by the `lib` here.

Also, perhaps the Credential ID should be something more detailed like `AttestationServerEmailPassword` to avoid conflicting with any other credentials.

Reply:
Oops, typo.